src/js/pm-image-editor.js

```diff
@@ -226,6 +226,25 @@
             };
 
             /**
+             * Update selection dimensions to new width and height.
+             * If width and height are correct -- reset all transformations to avoid positions issues.
+             *
+             * @param int width
+             * @param int height
+             */
+            ImageEditorFactory.prototype.updateSelectionDimensions = function(width, height) {
+                var parentSize = this.parentSize();
+                if (width <= parentSize.width && height <= parentSize.height) {
```

Comment:
If you do nothing in an error case, the invalid function call won't be noticed and will take longer to find. I'd prefer an error to be thrown.